.github/workflows/add_zip_to_release.yml

```
@@ -0,0 +1,51 @@
+      - name: Get release id
+        id: get_release_id
+        run: |
+          release_id=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+            "https://api.github.com/repos/${{ github.repository }}/releases/tags/${{ github.ref_name }}" \
+            | jq -r '.id // empty')
+          echo "Release ID: $release_id"
+          echo "release_id=$release_id" >> $GITHUB_ENV
+
+      - name: Generate final upload URL
+        id: generate_upload_url
+        run: |
```

actionlint (1.7.4)

property "create_release" is not defined in object type {get_release_id: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)